specs/~inheritance.yml

```diff
@@ -0,0 +1,225 @@
+overview: |
+  Like partials, Parent tags are used to expand an external template into the
+  current template. Contrary to partials, Parent tags may contain optional
+  arguments delimited by block tags. For this reason, Parent tags may also be
+  referred to as Parametric Partials.
+
+  The Parent tags' content MUST be a non-whitespace character sequence NOT
+  containing the current closing delimiter; each Parent tag MUST be followed by
+  an End Section tag with the same content within the matching Parent tag.
+
+  This tag's content names the Parent template to inject. Set Delimiter tags
+  MUST NOT affect the parsing of a Parent template. The Parent MUST be rendered
```

Comment:
"affect the parsing of a Parent template" feels awkward. It took me a couple of reads to figure out that we were saying Set Delimiter _outside_ the Parent tag shouldn't affect the template parsing of the partial.

I don't have recommended wording, other than maybe "the parsing of a Parent template" -> "the parsing of the injected template" feels a bit easier to grok?

Reply:
Good point, agreed. Will fix. I more or less copied this from the Partial overview.

Reply:
Yeah, I figured. The wording there somehow wasn't as ambiguous (probably because it didn't include the word "parent" to throw me off about which side was doing the delimiter setting).